gdiff '--color=auto' -urN o/install.cmake n/install.cmake
--- o/install.cmake	2025-06-09 21:42:29.000000000 +0200
+++ n/install.cmake	2025-09-28 13:43:19.221647900 +0200
@@ -1,45 +1,45 @@
 install(TARGETS ClamAV-GUI    
-    RUNTIME DESTINATION usr/bin/
+    RUNTIME DESTINATION bin/
 )
 
 install(FILES extra/icon16/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/16x16/apps/
+    DESTINATION share/icons/hicolor/16x16/apps/
 )
 
 install(FILES extra/icon22/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/22x22/apps/
+    DESTINATION share/icons/hicolor/22x22/apps/
 )
 
 install(FILES extra/icon32/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/32x32/apps/
+    DESTINATION share/icons/hicolor/32x32/apps/
 )
 
 install(FILES extra/icon48/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/48x48/apps/
+    DESTINATION share/icons/hicolor/48x48/apps/
 )
 
 install(FILES extra/icon64/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/64x64/apps/
+    DESTINATION share/icons/hicolor/64x64/apps/
 )
 
 install(FILES extra/icon128/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/128x128/apps/
+    DESTINATION share/icons/hicolor/128x128/apps/
 )
 
 install(FILES extra/icon256/clamav-gui.png
-    DESTINATION usr/share/icons/hicolor/256x256/apps/
+    DESTINATION share/icons/hicolor/256x256/apps/
 )
 
 install(DIRECTORY man 
-    DESTINATION usr/share/
+    DESTINATION share/
 )
 
 install(FILES CHANGES README
-    DESTINATION usr/share/doc/clamav-gui/
+    DESTINATION share/doc/clamav-gui/
 )
 
 install(FILES extra/ClamAV-GUI-ServiceMenu.desktop extra/ClamAV-GUI.desktop 
-    DESTINATION usr/share/applications/
+    DESTINATION share/applications/
 )
 
 install(FILES   translations/clamav-gui-uk_UA.qm
@@ -50,5 +50,5 @@
                 translations/clamav-gui-en_GB.qm
                 translations/clamav-gui-de_DE.qm
                 translations/clamav-gui-da_DK.qm
-    DESTINATION usr/share/clamav-gui/              
+    DESTINATION share/clamav-gui/              
 )
