diff -urN urtconnector.old/src/app_options.h urtconnector/src/app_options.h
--- urtconnector.old/src/app_options.h	2016-10-18 12:47:06.994895722 +0200
+++ urtconnector/src/app_options.h	2016-10-18 12:46:49.220513994 +0200
@@ -25,11 +25,14 @@
         ((start_hidden, bool, false)) \
         ((use_adv_cmd_line, bool, false)) \
         ((use_adv_cmd_line_42, bool, false)) \
+        ((use_adv_cmd_line_43, bool, false)) \
         ((adv_cmd_line, QString, QString())) \
         ((adv_cmd_line_42, QString, QString())) \
+        ((adv_cmd_line_43, QString, QString())) \
         ((separate_xsession, bool, true)) \
         ((binary_path, QString, QString())) \
         ((binary_path_42, QString, QString())) \
+        ((binary_path_43, QString, QString())) \
         ((geoip_database, QString, QString(URT_DATADIR"GeoIP.dat"))) \
         ((center_current_row, bool, true)) \
         ((keep_history, bool, true)) \
diff -urN urtconnector.old/src/filters/game_version_filter.cpp urtconnector/src/filters/game_version_filter.cpp
--- urtconnector.old/src/filters/game_version_filter.cpp	2016-10-18 12:47:06.994895722 +0200
+++ urtconnector/src/filters/game_version_filter.cpp	2016-10-18 12:54:58.182921187 +0200
@@ -13,6 +13,7 @@
 {
 const QString c_version_41("q3ut4");
 const QString c_version_42("q3urt42");
+const QString c_version_43("q3urt43");
 }
 
 ////////////////////////////////////////////////////////////////////////////////
@@ -44,7 +45,7 @@
 
 game_version_filter::game_version_filter(filter_class_p fc)
 : filter(fc)
-, version_(g_42)
+, version_(g_43)
 {
 }
 
@@ -59,6 +60,8 @@
             return si.game_type == c_version_41;
         case g_42:
             return si.game_type == c_version_42;
+        case g_43:
+            return si.game_type == c_version_43;
     }
     return false;
 }
@@ -120,6 +123,7 @@
 
     add_item(combo_, "4.1", g_41);
     add_item(combo_, "4.2", g_42);
+    add_item(combo_, "4.3", g_43);
 
     connect(f.get(), SIGNAL(changed_signal()), SLOT(filter_changed()));
     filter_changed();
diff -urN urtconnector.old/src/filters/game_version_filter.h urtconnector/src/filters/game_version_filter.h
--- urtconnector.old/src/filters/game_version_filter.h	2016-10-18 12:47:06.994895722 +0200
+++ urtconnector/src/filters/game_version_filter.h	2016-10-18 12:54:48.923931515 +0200
@@ -11,7 +11,8 @@
 enum game_version_t
 {
     g_41,
-    g_42
+    g_42,
+    g_43
 };
 
 class game_version_filter_quick_opt_widget : public QWidget
diff -urN urtconnector.old/src/main_window.cpp urtconnector/src/main_window.cpp
--- urtconnector.old/src/main_window.cpp	2016-10-18 12:47:06.995895181 +0200
+++ urtconnector/src/main_window.cpp	2016-10-18 12:49:06.577186015 +0200
@@ -589,13 +589,25 @@
     }
 #endif
 
+    bool is_urt43 = info->game_type == "q3urt43";
     bool is_urt42 = info->game_type == "q3urt42";
 
-    QString binary_path = is_urt42 ? as.binary_path_42() : as.binary_path();
-    QString adv_cmd_line = is_urt42 ? (as.use_adv_cmd_line_42() ? as.adv_cmd_line_42() : QString())
-                                    : (as.use_adv_cmd_line() ? as.adv_cmd_line() : QString()) ;
-
-    QString version = is_urt42 ? "4.2" : "4.1";
+    QString binary_path = as.binary_path();
+    QString adv_cmd_line = as.use_adv_cmd_line() ? as.adv_cmd_line() : QString();
+    QString version = "4.1";
+    
+    if (is_urt43)
+    {
+        QString binary_path = as.binary_path_43();
+        QString adv_cmd_line = as.use_adv_cmd_line_43() ? as.adv_cmd_line_43() : QString();
+        QString version = "4.3"; 
+    } 
+    else if (is_urt42)
+    {
+        QString binary_path = as.binary_path_42();
+        QString adv_cmd_line = as.use_adv_cmd_line_42() ? as.adv_cmd_line_42() : QString();
+        QString version = "4.2"; 
+    }
 
     if (binary_path.isEmpty() || !QFileInfo(binary_path).exists() || !QFileInfo(binary_path).isExecutable())
     {
@@ -618,7 +630,9 @@
                 continue;
             }
 
-            if (is_urt42)
+            if (is_urt43)
+                as.binary_path_43_set(binary_path);
+            else if (is_urt42)
                 as.binary_path_42_set(binary_path);
             else
                 as.binary_path_set(binary_path);
diff -urN urtconnector.old/src/server_info_manager.cpp urtconnector/src/server_info_manager.cpp
--- urtconnector.old/src/server_info_manager.cpp	2016-10-18 12:47:06.995895181 +0200
+++ urtconnector/src/server_info_manager.cpp	2016-10-18 12:50:12.998243510 +0200
@@ -280,7 +280,12 @@
     server_info += tr("<tr class=\"line1\"><td>Country</td><td>%1 %2</td></tr>").arg(country_flag).arg(si.country);
     server_info += tr("<tr class=\"line2\"><td>Public slots</td><td>%1</td></tr>").arg(si.max_player_count - si.private_slots());
     server_info += tr("<tr class=\"line1\"><td>Total slots</td><td>%1</td></tr>").arg(si.max_player_count);
-    QString version = si.game_type == "q3urt42" ? "4.2" : "4.1";
+    QString version = "4.1";
+    if (si.game_type == "q3urt43")
+        QString version = "4.3";
+    else if (si.game_type == "q3urt42")
+        QString version = "4.2";
+
     server_info += tr("<tr class=\"line2\"><td>Game version</td><td>%1</td></tr>").arg(version);
     
     if ( !si.forbidden_gears().empty() )
diff -urN urtconnector.old/src/setting_forms/launch_settings_form.cpp urtconnector/src/setting_forms/launch_settings_form.cpp
--- urtconnector.old/src/setting_forms/launch_settings_form.cpp	2016-10-18 12:47:06.995895181 +0200
+++ urtconnector/src/setting_forms/launch_settings_form.cpp	2016-10-18 12:46:49.221513453 +0200
@@ -32,6 +32,7 @@
 
     connect(p_->ui.urt41_panel, SIGNAL(changed()), SIGNAL(changed()));
     connect(p_->ui.urt42_panel, SIGNAL(changed()), SIGNAL(changed()));
+    connect(p_->ui.urt43_panel, SIGNAL(changed()), SIGNAL(changed()));
 
     connect( p_->ui.x_check_button, SIGNAL(clicked()), this, SLOT( x_check() ));
     connect( p_->ui.select_mumble_bin_button, SIGNAL(clicked()), this, SLOT( choose_mumble_bin() ));
@@ -54,12 +55,16 @@
 {
     launch_opts_panel* p41 = p_->ui.urt41_panel;
     launch_opts_panel* p42 = p_->ui.urt42_panel;
+    launch_opts_panel* p43 = p_->ui.urt43_panel;
     p41->set_separate_xsession(p_->ui.separate_x_check->isChecked());
     p41->set_use_mumble_overlay(p_->ui.mumble_overlay_group->isChecked());
     p41->set_mumble_overlay_bin(p_->ui.mumble_bin_edit->text());
     p42->set_separate_xsession(p_->ui.separate_x_check->isChecked());
     p42->set_use_mumble_overlay(p_->ui.mumble_overlay_group->isChecked());
     p42->set_mumble_overlay_bin(p_->ui.mumble_bin_edit->text());
+    p43->set_separate_xsession(p_->ui.separate_x_check->isChecked());
+    p43->set_use_mumble_overlay(p_->ui.mumble_overlay_group->isChecked());
+    p43->set_mumble_overlay_bin(p_->ui.mumble_bin_edit->text());
     if (!lock_change_)
         emit changed();
 }
@@ -72,13 +77,17 @@
 
     launch_opts_panel* p41 = p_->ui.urt41_panel;
     launch_opts_panel* p42 = p_->ui.urt42_panel;
+    launch_opts_panel* p43 = p_->ui.urt43_panel;
 
     p41->set_binary_path(as.binary_path());
     p42->set_binary_path(as.binary_path_42());
+    p43->set_binary_path(as.binary_path_43());
     p41->set_adv_cmd_line( as.adv_cmd_line() );
     p42->set_adv_cmd_line( as.adv_cmd_line_42() );
+    p43->set_adv_cmd_line( as.adv_cmd_line_43() );
     p41->set_use_adv_cmd_line( as.use_adv_cmd_line() );
     p42->set_use_adv_cmd_line( as.use_adv_cmd_line_42() );
+    p43->set_use_adv_cmd_line( as.use_adv_cmd_line_43() );
     p_->ui.update_server_check->setChecked( as.update_before_connect() );
     p_->ui.multiple_launch_check->setChecked( as.multiple_launch() );
     p_->ui.after_launch_combo->setCurrentIndex( as.after_launch_action() );
@@ -86,14 +95,17 @@
     p_->ui.separate_x_check->setChecked( as.separate_xsession() );
     p41->set_separate_xsession( as.separate_xsession() );
     p42->set_separate_xsession( as.separate_xsession() );
+    p43->set_separate_xsession( as.separate_xsession() );
 
     p_->ui.mumble_overlay_group->setChecked( as.use_mumble_overlay() );
     p41->set_use_mumble_overlay( as.use_mumble_overlay() );
     p42->set_use_mumble_overlay( as.use_mumble_overlay() );
+    p43->set_use_mumble_overlay( as.use_mumble_overlay() );
 
     p_->ui.mumble_bin_edit->setText( as.mumble_overlay_bin() );
     p41->set_mumble_overlay_bin( as.mumble_overlay_bin() );
     p42->set_mumble_overlay_bin( as.mumble_overlay_bin() );
+    p43->set_mumble_overlay_bin( as.mumble_overlay_bin() );
 #endif
 }
 
@@ -103,13 +115,17 @@
 
     launch_opts_panel* p41 = p_->ui.urt41_panel;
     launch_opts_panel* p42 = p_->ui.urt42_panel;
+    launch_opts_panel* p43 = p_->ui.urt43_panel;
 
     as.binary_path_set( p41->binary_path() );
     as.binary_path_42_set( p42->binary_path() );
+    as.binary_path_43_set( p43->binary_path() );
     as.use_adv_cmd_line_set( p41->use_adv_cmd_line() );
     as.use_adv_cmd_line_42_set( p42->use_adv_cmd_line() );
+    as.use_adv_cmd_line_43_set( p43->use_adv_cmd_line() );
     as.adv_cmd_line_set( p41->adv_cmd_line() );
     as.adv_cmd_line_42_set( p42->adv_cmd_line() );
+    as.adv_cmd_line_43_set( p43->adv_cmd_line() );
     as.update_before_connect_set(p_->ui.update_server_check->isChecked());
     as.multiple_launch_set(p_->ui.multiple_launch_check->isChecked());
     as.after_launch_action_set(p_->ui.after_launch_combo->currentIndex());
@@ -132,10 +148,13 @@
     app_settings as;
     as.binary_path_reset();
     as.binary_path_42_reset();
+    as.binary_path_43_reset();
     as.use_adv_cmd_line_reset();
     as.use_adv_cmd_line_42_reset();
+    as.use_adv_cmd_line_43_reset();
     as.adv_cmd_line_reset();
     as.adv_cmd_line_42_reset();
+    as.adv_cmd_line_43_reset();
     as.update_before_connect_reset();
     as.multiple_launch_reset();
     as.after_launch_action_reset();
diff -urN urtconnector.old/src/setting_forms/launch_settings_form.ui urtconnector/src/setting_forms/launch_settings_form.ui
--- urtconnector.old/src/setting_forms/launch_settings_form.ui	2016-10-18 12:47:06.995895181 +0200
+++ urtconnector/src/setting_forms/launch_settings_form.ui	2016-10-18 12:46:49.221513453 +0200
@@ -156,6 +156,19 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="tab_3">
+      <attribute name="title">
+       <string>UrT 4.3</string>
+      </attribute>
+      <layout class="QVBoxLayout" name="verticalLayout_3">
+       <property name="margin">
+        <number>0</number>
+       </property>
+       <item>
+        <widget class="launch_opts_panel" name="urt43_panel" native="true"/>
+       </item>
+      </layout>
+     </widget>
     </widget>
    </item>
   </layout>
