From ef009ee0df12b0e6b0f1c4d07809c9c11a588f54 Mon Sep 17 00:00:00 2001
From: Stricted <info@stricted.net>
Date: Thu, 18 Aug 2016 00:48:00 +0200
Subject: [PATCH] fix bug#48254

---
 py/jk_update.in | 25 +++++++++++++++++++++----
 1 file changed, 21 insertions(+), 4 deletions(-)

diff --git a/py/jk_update.in b/py/jk_update.in
index eafc2f0..afc930b 100644
--- a/py/jk_update.in
+++ b/py/jk_update.in
@@ -112,6 +112,9 @@ def find2update(jail, dir, skips, config, files2update=[],files2clean=[]):
 						files2clean.append(dir+file) 
 			except OSError, (errno,strerror):
 				if (errno == 2):
+					if (os.path.isdir(jail+dir+file)):
+						files2update, files2clean = find2update(jail, dir+file+'/', skips, config, files2update, files2clean)
+					
 					files2clean.append(dir+file)
 				else:
 					sys.stderr.write('ERROR: while checking if '+jail+dir+file+' needs to be updated: '+strerror+'\n')
@@ -133,6 +136,7 @@ def updatejail(jail, dirs, skips, config):
 		else:
 			files = []
 			cleans = []
+			cdirs = []
 			try:
 				files,cleans = find2update(jail, dir, skips, config, [],[])
 			except OSError, (errno,strerror):
@@ -151,12 +155,25 @@ def updatejail(jail, dirs, skips, config):
 				if (config['dry-run'] == 1):
 					allcleans.append(file)
 				else:
-					print 'removing deprecated file '+jail+file
+					if (os.path.isdir(jail+file)):
+						cdirs.append(file)
+					else:
+						print 'removing deprecated file '+jail+file
+						try:
+							os.unlink(jail+file)
+							allcleans.append(file)
+						except:
+							sys.stderr.write('ERROR: failed to remove deprecated file '+jail+file+'\n')
+			for cdir in cdirs:
+				if (config['dry-run'] == 1):
+					allcleans.append(cdir)
+				else:
+					print 'removing deprecated directory '+jail+file
 					try:
-						os.unlink(jail+file)
-						allcleans.append(file)
+						os.rmdir(jail+cdir)
+						allcleans.append(cdir)
 					except:
-						sys.stderr.write('ERROR: failed to remove deprecated file '+jail+file+'\n')
+						sys.stderr.write('ERROR: failed to remove deprecated directory '+jail+file+'\n')
 	if (config['dry-run'] == 1):
 		for file in allfiles:
 			print 'will update outdated file '+jail+file
-- 
2.9.0.windows.1

