diff --git a/py/Makefile.in b/py/Makefile.in
--- a/py/Makefile.in
+++ b/py/Makefile.in
@@ -50,10 +50,10 @@
 	    -e "s!INIPREFIX='/etc/jailkit'!INIPREFIX='${INIPREFIX}'!" \
 	    -e "s!LIBDIR='[a-z/]*'!LIBDIR='${PYLIBDIR}'!" \
 	    -e "s!EXEPREFIX='[a-z/]*'!EXEPREFIX='${PREFIX}'!" \
-	    -e "s:#!/usr/bin/python:#!$(PYTHONINTERPRETER):" < $< > $@
+	    -e "s:#!/usr/bin/python:#!${PYTHON}:" < $< > $@
 
 .py.pyc:
-	$(PYTHONINTERPRETER) -c "import py_compile;py_compile.compile('$<')"
+	${PYTHON} -c "import py_compile;py_compile.compile('$<')"
 
 PY_BINS = jk_cp jk_init jk_check jk_addjailuser jk_jailuser jk_list jk_update
 PY_LIBS = jk_lib.pyc

