--- Makefile.o	2019-08-21 21:54:55.545022530 +0200
+++ Makefile	2019-08-21 21:55:11.663343524 +0200
@@ -39,14 +39,14 @@
 GO_WHATSAPP_DATE =$(shell $(GIT) --git-dir="$(GO_WHATSAPP_GIT)" log -1 --date=rfc --format=%cd)
 
 update-dep:
-	$(GO) get -u github.com/Rhymen/go-whatsapp
+#	$(GO) get -u github.com/Rhymen/go-whatsapp
 	touch -d "$(GO_WHATSAPP_DATE)" $(GO_WHATSAPP_A)
 
 $(GO_WHATSAPP_A):
-	$(GO) get github.com/Rhymen/go-whatsapp
+#	$(GO) get github.com/Rhymen/go-whatsapp
 
 purplegwa.a: purplegwa.go purplegwa-media.go $(GO_WHATSAPP_A)
-	$(GO) get github.com/skip2/go-qrcode
+#	$(GO) get github.com/skip2/go-qrcode
 	$(GO) build -buildmode=c-archive -o purplegwa.a purplegwa.go purplegwa-media.go
 
 $(TARGET): $(PURPLE_C_FILES) $(PURPLE_COMPAT_FILES) purplegwa.a
