diff '--color=auto' -urN o/olefy.py n/olefy.py
--- o/olefy.py	2025-07-25 18:50:56.613391948 +0200
+++ n/olefy.py	2025-07-25 18:52:39.666640277 +0200
@@ -37,7 +37,7 @@
 olefy_listen_port = int(os.getenv('OLEFY_BINDPORT', '10050'))
 olefy_tmp_dir = os.getenv('OLEFY_TMPDIR', '/tmp')
 olefy_python_path = os.getenv('OLEFY_PYTHON_PATH', '/usr/bin/python3')
-olefy_olevba_path = os.getenv('OLEFY_OLEVBA_PATH', '/usr/local/bin/olevba3')
+olefy_olevba_path = os.getenv('OLEFY_OLEVBA_PATH', '/usr/bin/olevba')
 # 10:DEBUG, 20:INFO, 30:WARNING, 40:ERROR, 50:CRITICAL
 olefy_loglvl = int(os.getenv('OLEFY_LOGLVL', 20))
 olefy_min_length = int(os.getenv('OLEFY_MINLENGTH', 500))
diff '--color=auto' -urN o/olefy.conf n/olefy.conf
--- o/olefy.conf	2025-07-25 18:54:57.882923725 +0200
+++ n/olefy.conf	2025-07-25 18:55:19.628132990 +0200
@@ -12,7 +12,7 @@
  # no command line options allowed here
 OLEFY_PYTHON_PATH=/usr/bin/python3
  # no command line options allowed here
-OLEFY_OLEVBA_PATH=/usr/local/bin/olevba
+OLEFY_OLEVBA_PATH=/usr/bin/olevba
 
  # 10:DEBUG, 20:INFO, 30:WARNING, 40:ERROR, 50:CRITICAL
 OLEFY_LOGLVL=30
