--- src/TagLib/Ogg/XiphComment.cs.old	2015-02-27 23:19:23.702925142 +0100
+++ src/TagLib/Ogg/XiphComment.cs	2015-02-27 23:22:06.945160011 +0100
@@ -892,18 +892,18 @@
 		///    current instance, or zero if not specified.
 		/// </value>
 		/// <remarks>
-		///    This property is implemented using the "TEMPO" field.
+		///    This property is implemented using the "BPM" field.
 		/// </remarks>
 		public override uint BeatsPerMinute {
 			get {
-				string text = GetFirstField ("TEMPO");
+				string text = GetFirstField ("BPM");
 				double value;
 				return (text != null &&
 					double.TryParse (text, out value) &&
 					value > 0) ? (uint) Math.Round (value) :
 					0;
 			}
-			set {SetField ("TEMPO", value);}
+			set {SetField ("BPM", value);}
 		}
 		
 		/// <summary>
